_sass/index.scss

```diff
@@ -334,7 +413,7 @@
       }
 
       .sponsor-tier:not(:last-child) {
-        border-bottom: 1px solid $hp-pink-1;
+        border-bottom: 1px solid $hp-brown-1;
```

Comment:
These dividing seem a little too aggressive, maybe try a lighter shade? There's no design in the spec for this section, but there's also no other place in the design where these dividers are shown.

Reply:
I switched to orange. PTAL